src/Microsoft.DotNet.Build.Tasks/GetPackageVersion.cs

```diff
@@ -27,42 +26,19 @@ public override bool Execute()
         {
             Log.LogMessage("Starting GetPackageVersion");
 
-            using (var repository = new Repository(RepositoryRoot))
```

Comment:
With this change I assume we can remove the dependency on the git library now correct?